internal/characters/ayato/burst.go

```diff
@@ -75,7 +75,7 @@ func (c *char) Burst(p map[string]int) action.ActionInfo {
 				return
 			}
 			//deal dmg
-			c.Core.QueueAttackWithSnap(ai, snap, combat.NewCircleHit(c.Core.Combat.Player(), 9), 0)
+			c.Core.QueueAttackWithSnap(ai, snap, combat.NewCircleHit(c.Core.Combat.PrimaryTarget(), 2.5), 0)
```

Comment:
should I change `... range c.Core.Combat.Enemies()` into EnemiesWithinRadius? What's the radius ayato burst uses to select targets? (also same for Ganyu)

Reply:
I think the targeting for ayato and ganyu burst is not as simple as supplying `target` because sometimes the droplets/icicles do not have a target to center on, they get placed randomly which happens to clip a target

Reply:
The actual targeting logic fix is out of scope for this PR imo, this is just to improve the current situation (it found a target so it should center on it). pls revert your changes to the target == -1 if check in both ayato's and ganyu's PR

Reply:
alright. I still can't only supply c.Core.Combat.Enemy(target) to their bursts, because when target == -1 but the Rand.Float64() < prob, it will crash trying to center on a -1 target.

Reply:
Ah I see, keep it then